tailwind.config.js

```diff
@@ -5,10 +5,11 @@ module.exports = {
     "./components/**/*.{js,ts,jsx,tsx}",
   ],
   theme: {
-    colors: {
-      gray: "#222322",
-      white: "#ffffff",
-      blue: "#338ee8",
+    extend: {
+      colors: {
+        CYgray: "#222322",
```

Comment:
It might make sense to call this cy-gray instead of camel case, as it'll match the rest of the classes better

Reply:
Linter screams at me if i try, camel case wasn't my first pick unfortunately